src/coreclr/jit/gentree.cpp

```diff
@@ -19234,6 +19239,18 @@ bool GenTree::isEvexCompatibleHWIntrinsic()
 #endif
 }
 
+//------------------------------------------------------------------------
+// isEmbBroadcastHWIntrinsic: Checks if the intrinsic is a embedded broadcast compatible inintrsic.
+//
+// Return Value:
+// true if the intrisic node lowering instruction is embedded broadcast compatible.
+//
+bool GenTree::isEmbBroadcastHWIntrinsic()
+{
+    assert(gtOper == GT_HWINTRINSIC);
+    return HWIntrinsicInfo::IsEmbBroadcastCompatible(AsHWIntrinsic()->GetHWIntrinsicId());
+}
```

Comment:
Given the assert, it'd probably be better to expose this directly on `GenTreeHWIntrinsic` instead. That helps enforce correctness.